wordle-web/src/views/game.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+    <div class="game">
+        <v-card>
+            <v-card-title> 'a clever name for the game' or something </v-card-title>
+            <v-card-text>
+                This is a version of the popular game Wordle, eventually.
```

Comment:
Why is v-card-title on one line and v-card-text on three?

Reply:
A title is consistently just one line of text, whereas the text field is meant to support blocks of text so this better represents each of their purposes and functionalities, rather than having to reorganize how the lines are formatted if we wanted to add another line of text to the text block for example.